src/dbnode/storage/types.go

```diff
@@ -1404,7 +1405,25 @@ type TileAggregator interface {
 }
 
 // NewTileAggregatorFn creates a new TileAggregator.
-type NewTileAggregatorFn func(iOpts instrument.Options) TileAggregator
+type NewTileAggregatorFn func(opts TileAggregatorOptions) TileAggregator
+
+// TileAggregatorOptions represents the options for tile aggregation.
```

Comment:
Micro nit: 
```suggestion
// TileAggregatorOptions represents the options for TileAggregator.
```
(at least in Goland, this gives nice clickable hyperlinks to navigate the source code)